plugins/common/shim/processor.go

```diff
@@ -52,7 +52,7 @@ func (s *Shim) RunProcessor() error {
 	for {
 		m, err := parser.Next()
 		if err != nil {
-			if err == influx.EOF {
+			if err == influx.ErrEOF {
```

Comment:
Can we please use `errors.Is(err, influx.ErrEOF)` to gracefully handle wrapped errors?